### PR TITLE
Keyboard Extensions Docs

### PR DESCRIPTION
**Pull request by PureWeen**

### CODE REVIEW

File: docs/maui/extensions/keyboard-extensions.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: KeyboardExtensions - .NET MAUI Community Toolkit
+author: PureWeen
+description: "The KeyboardExtensions provide a series of extension methods that support interacting with the Keyboard."
+ms.date: 04/12/2023
+---
+
+# KeyboardExtensions
+
+The `KeyboardExtensions` provide a series of extension methods that support interacting with the Keyboard..
```
Comment (pictos) [inline docs/maui/extensions/keyboard-extensions.md:10]:
```suggestion
The `KeyboardExtensions` provide a series of extension methods that support interacting with the Keyboard.
```

File: docs/maui/extensions/keyboard-extensions.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Soft Keyboard Methods
+
+The following methods allow you to close, show, and check if the keyboard is open.
+
+### Hide Keyboard
+
+The `HideKeyboardAsync` method will hide the Soft Input Keyboard if it is currently visible
+
+The following example shows how to hide the keyboard:
```
Comment (pictos) [inline docs/maui/extensions/keyboard-extensions.md:26]:
```suggestion
The following example shows how to hide the keyboard for a given entry:
```

File: docs/maui/extensions/keyboard-extensions.md (hunk 1 above)
Comment (pictos) [inline docs/maui/extensions/keyboard-extensions.md:10]: Should we mention that those methods can be used for `InputView` elements?

**Review by pictos — COMMENTED** [review]
It looks good for me, just some points

File: docs/maui/extensions/keyboard-extensions.md (hunk 1 above)
Comment (PureWeen) [reply docs/maui/extensions/keyboard-extensions.md]: updated
Let me know if that sounds alright?

Comment (pictos) [reply docs/maui/extensions/keyboard-extensions.md]: sounds amazing <3

**Review by pictos — APPROVED** [review]
LGTM

**Review by jfversluis — APPROVED** [review]

**Pull request merged**